draft-ietf-jsonpath-base.md

```diff
@@ -227,6 +227,11 @@ Normalized Path:
   providing a query that results in exactly that node.  Similar
   to, but syntactically different from, a JSON Pointer {{-pointer}}.
 
+Unicode Scalar Value:
+: Any Unicode code point except high-surrogate and low-surrogate code points.
+  In other words, base 16 integers in either of the inclusive ranges 0 to D7FF and
+  E000 to 10FFFF. JSON values of type string are sequences of Unicode scalar values.
```

Comment:
Maybe add a reference to Unicode, I provided the locational stuff in one of my comments on the issue.

Reply:
Added a reference to the latest Unicode standard (v14.0) since this supersedes earlier versions. (I couldn't find your locational comment.) Not sure if there is a slicker way to define this reference - @cabo?